lib/utils/Decimals.ts

```diff
@@ -2,6 +2,10 @@ import { BigDecimal, BigInt } from '@graphprotocol/graph-ts';
 
 export const DEFAULT_DECIMALS = 18;
 
+export const BIG_DECIMAL_1E9 = BigDecimal.fromString('1e9') 
```

Comment:
Can we add in a zero value BigDecimal for ease of returning zero values for reverted calls?

Reply:
I am doing a reverted check before proceeding with call.value, something like this:
`    let reservesCall = pair.try_getReserves()
    if (reservesCall.reverted) {
        return BigDecimal.zero()
    }
    ...
    `